src/view/com/util/List.tsx

```diff
@@ -37,7 +40,8 @@ function ListImpl<ItemT>(
   const isScrolledDown = useSharedValue(false)
   const contextScrollHandlers = useScrollHandlers()
   const pal = usePalette('default')
-
+  const showsVerticalScrollIndicator =
+    useGate('shows_vertical_scroll_indicator') && isWeb
```

Comment:
I am using `shows_vertical_scroll_indicator` for the gate name, even though we're actually _removing_ the indicators. I think it makes more sense to use the wording `shows_vertical_scroll_indicator` just so we are not inverting the boolean. We can change the name though if we want to keep the gate name identical to what we are actually testing.

Reply:
Ah, I just realized this isn't ideal because the `useGate` will return `false` IIRC.